lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -130,6 +136,8 @@ class PatchEntryPointMutate : public Patch {
     SmallVector<UserDataNodeUsage, 8> rootDescriptors;
     // Per-descriptor-set lists of lgc.descriptor.set calls
     SmallVector<UserDataNodeUsage, 8> descriptorSets;
+    // Per-UserDataMapping lists of lgc.special.user.data calls
+    SmallVector<UserDataNodeUsage, 8> specialUserData;
```

Comment:
I'd suggest sizing this to 18, which is how many relevant enums we currently have.